airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -16,94 +16,147 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-"""
-This module contains integration with Azure Blob Storage.
+"""This module contains integration with Azure Blob Storage."""
 
-It communicate via the Window Azure Storage Blob protocol. Make sure that a
-Airflow connection of type `wasb` exists. Authorization can be done by supplying a
-login (=Storage account name) and password (=KEY), or login and SAS token in the extra
-field (see connection `wasb_default` for an example).
```

Comment:
This doc seems useful -- does it exist elsewhere?

Reply:
Ah.. mistake, I'll add it back